flambe/metric/dev/perplexity.py

```diff
@@ -28,3 +30,42 @@ def compute(self, pred: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
         """
         entropy = self.entropy(pred, target).mean()
         return torch.exp(entropy)
+
+    def aggregate(self, state: dict, *args, **kwargs) -> Dict:
+        """Aggregates by only storing entropy per sample
+
+        Parameters
+        ----------
+        state: dict
+            the metric state
+        args: the pred, target tuple
+
+        Returns
+        -------
+        dict
+            the state dict
+        """
+        pred, target = args
+        if not state:
+            state['historic_score'] = []
+        state['historic_score'].append(self.entropy(pred, target).cpu().detach())
+        return state
+
+    def finalize(self, state: Dict) -> float:
+        """Finalizes the metric computation
+
+        Parameters
+        ----------
+        state: dict
+            the metric state
+
+        Returns
+        -------
+        float
+            The final score.
```

Comment:
make sure to leave a blank line at the end of the docstring

Reply:
done